css/style.css

```diff
@@ -18,6 +10,57 @@ a {
   color: #1f2328;
 }
 
+a {
+  text-decoration: none;
+}
+
+.flex {
+  display: flex;
+}
+
+li {
+  list-style: none;
+}
+
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the navigation bar :+1: :sunglasses: However it would be nice to change the cursor (`cursor: pointer`) when user passes the cursor over the navigation links or what do you think ?